identity-service/src/app.js

```diff
@@ -72,7 +73,7 @@ class App {
     }
 
     try {
-      await txRelay.fundEthRelayerIfEmpty()
```

Comment:
@hareeshnagaraj Did your mean to delete this line?

Reply:
no shit